api/src/pcapi/workers/fully_sync_venue_job.py

```diff
@@ -0,0 +1,10 @@
+from pcapi.core.offerers import models as offerers_models
+from pcapi.scripts.stock.fully_sync_venue import fully_sync_venue
+from pcapi.workers import worker
+from pcapi.workers.decorators import job
+
+
+@job(worker.low_queue)
+def fully_sync_venue_job(venue_id: int) -> None:
+    venue = offerers_models.Venue.query.filter(offerers_models.Venue.id == venue_id)
```

Comment:
Il semble manquer le `.one()`, ou plus simplement : 
```suggestion
    venue = offerers_models.Venue.query.get(venue_id)
```